packages/rest/src/client/AbstractRestClient.ts

```diff
@@ -247,7 +247,7 @@ export abstract class AbstractRestClient {
             /**
              * Perform the actual http request
              */
-            let clientRequest: https.ClientRequest | http.ClientRequest;
+            let clientRequest: http.ClientRequest;
```

Comment:
I'm a little confused as to why this got reduced to only be an instance of http.ClientRequest when it should depend on the protocol specified in the session creation?

Reply:
This confused me too at first :) The `@types/node` package has changed to expect an instance of the same type `http.ClientRequest`, for both HTTP and HTTPS requests. @dkelosky made the same change in the "next" branch.